contents/blog/what-you-need-to-know-about-open-source-analytics.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2021-11-29
+title: What you need to know about open-source analytics
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["vivek-sonar"]
+featuredImage: ../images/blog/posthog-blog-image.png
+featuredImageType: standard
+---
+
+When you imagine product analytics, you may think of tools built by and for large companies, such as Google Analytics or Amplitude. But there are also many awesome open-source tools which are created in a different way and that can be useful for small businesses, freelancers, and individuals too. These open-source tools often don’t require large up-front costs or lengthy contracts and some are available for free. 
+
+In this article we’ll explain what product analytics is, what it means for it to be open source and what the advantages are for users in teams of all sizes. 
+
+## What is product analytics, and why is it used?
+Product analytics is the method of collecting and investigating product data in order to gain insight. You may use these insights to make better decisions about a process or make an existing design more efficient. You may use them to change the type of users you are targeting, or to change your pricing model. Either way, these insights enable you to build better products, faster. 
+
+A typical example of product analytics in action is using a tool such as PostHog to [track users through a funnel in your product or website](/docs/user-guides/funnels), so you can make improvements that increase customer activation. Companies such as [Hasura have used PostHog to improve conversion by 10-20%](/customers/hasura) by doing exactly this!
+
+Product analytics is useful for many different teams within an organization, from marketing to finance, and can be used in any type of product, from [games](/blog/building-the-future-of-game-analytics-pureskill) to [productivity software](/customers/saga). Teams can use product analytics to answer questions such as: 
+
+- What content should I create? 
+- What should I change to make my site more accessible and navigable? 
+- How can I improve my conversion rate or increase my earnings? 
+- How do users discover my product?
+- How do users react to our pricing?
+
+## What is open source software?
+Open source is a licensing model which enables anyone to see and modify the source code within the software. This isn’t the case for most software, which is closed source. There are a variety of open-source licenses, but most open source projects are also distributed for free - this is called Free and Open Source Software (FOSS). 
+
+PostHog is an example of an open-source project, as anyone can access our source code via our GitHub repo and we offer a FOSS version of PostHog under a permissive MIT license. This means that the only restriction is that a license and copyright notice must be included with all versions of the software.
+
+## Why is being open-source important in product analytics?
```

Comment:
I think a very important point missing here is on data ownership & control and end-user privacy.

Reply:
Have pushed a commit about this, thanks Paolo!